chart/compass/templates/director-auth-test.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: "testing.kyma-project.io/v1alpha1"
 kind: TestDefinition
 metadata:
-  name: {{ .Chart.Name }}-e2e
+  name: {{ .Chart.Name }}-director-auth-related-tests
```

Comment:
Hmm, we have there tests for:
- different auth methods
- accessing playground
- tenant mapping handler
- viewer query

Basically, these are tests, that are executed through gateway. So, what do you think about the name `-director-gateway-integration` or something like that?